networkx/linalg/bethehessianmatrix.py

```diff
@@ -35,7 +35,7 @@ def bethe_hessian_matrix(G, r=None, nodelist=None):
 
     Returns
     -------
-    H : Numpy matrix
+    H : scipy.sparse.csr_matrix
```

Comment:
I like this level of specificity.  Should the same be used above, instead of simply "SciPy sparse matrix"?  Or were you keeping your options open there, in case it might change in the future?